tools/apksize/apksize.gradle

```diff
@@ -50,6 +51,36 @@ task uploadMeasurements(type: UploadMeasurementsTask) {
     uploader = "https://storage.googleapis.com/firebase-engprod-metrics/upload_tool.jar"
 }
 
+// TODO(yifany@): Move below 2 tasks out of apksize.gradle.
+task generateCoverageMeasurements(type: GenerateCoveragePercentsTask, group: 'Coverage') {
+
+    // TODO(yifany@): Find a more concise way to express dependencies.
+    dependsOn ':firebase-common:checkCoverage'
```

Comment:
You can achieve this with the Gradle API.

See https://docs.gradle.org/current/dsl/org.gradle.api.Project.html and https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:subprojects(groovy.lang.Closure)

You can also filter the project in the closure so you don't apply this to the tools, buildSrc, etc projects.

Reply:
Done.